enterprise/frontend/src/metabase-enterprise/whitelabel/components/IllustrationWidget/IllustrationWidget.tsx

```diff
@@ -140,15 +146,15 @@ export function IllustrationWidget({
             data={data}
             value={value}
             onChange={handleChange}
-            w={isCustomIllustration && fileName ? "8.25rem" : "100%"}
+            w={isCustomIllustration ? "8.25rem" : "100%"}
```

Comment:
don't worry about the style too much. It will be changed soon to match [the new design.](https://www.figma.com/file/yOksAHQ5ih6ospZd6f3RVc/Let-white-labeling-customers-customize-illustrations?type=design&node-id=1486-122158&mode=design&t=ieKLYahwWumPT4eu-0)